package.json

```diff
@@ -31,6 +29,7 @@
   "scripts": {
     "build": "rollup --config",
     "build-watch": "rollup --watch --config",
+		"eslint": "eslint .",
```

Comment:
I'm not sure if this is needed because the IDE runs eslint automatically; incorrect indentation on this line

Reply:
Yeah not sure why things got messed up like that... Cause it doesn't look that way on my side. :(  It's handy to be able to run eslint and find all the errors and fix them across the codebase before commits / etc when you don't see them in the IDE. I didn't have the eslint file in the right place until the very end, so had a little cleanup to do in one or two of the last commits. Copied the additions to eslint that you posted in Discord.